### ISSUE TITLE
Firefox: Hitting save when a tunnel is assigned to an interface will result in a validation error.

### ISSUE THREAD

**Issue by FineWolf** [theonemcdonald/pfSense-pkg-WireGuard#23]
When a WireGuard Tunnel is assigned to an interface, it cannot be disabled.
This currently results in the checkbox being disabled on the frontend.

However, under Firefox, `<input type="checkbox" disabled>` do not send their value on FORM POST (this is the actual proper behavior according to the [HTML specification](https://www.w3.org/TR/REC-html40-971218/interact/forms.html#h-17.12.1)).

This leads to a validation error when the form gets validated on save.

To fix, three possible solutions:

- Remove the checkbox entirely, replacing it with a `<input type="hidden" value="yes" name="enabled" />`; while keeping the help text stating that the tunnel cannot be disabled.
- Remove [the validation](https://github.com/theonemcdonald/pfSense-pkg-WireGuard/blob/9858a0181861ac1edbfacd8a769c947682d424eb/src/files/usr/local/pkg/wireguard/wg_validate.inc#L73) for that specific field and just completely ignore the value when the tunnel is assigned to an interface.
- Force that value to be `yes` before validation when the form gets posted, ignoring what's included in the POST request.

**Comment by theonemcdonald** [comment]
What version are you running? I caught this and added a hidden field 

https://github.com/theonemcdonald/pfSense-pkg-WireGuard/blob/main/src/files/usr/local/www/wg/vpn_wg_edit.php#L180

**Comment by theonemcdonald** [comment]
@FineWolf Try the latest pre-compiled build v0.0.5_1. This issue should be resolved.

**Comment by FineWolf** [comment]
```
pfSense-pkg-WireGuard-0.0.5    pfSense package WireGuard
wireguard-kmod-0.0.20210412    WireGuard implementation for the FreeBSD kernel
wireguard-tools-1.0.20210315_4 Fast, modern and secure VPN Tunnel
```

On pfSense+ 21.02.2-RELEASE.
Will update and report back.

**Comment by FineWolf** [comment]
Resolved in 0.0.5_1. Thanks!
